os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -445,7 +445,7 @@ int8_t cmd_wifi_connect(trwifi_ap_config_s *ap_connect_config, void *arg)
 		password = ap_connect_config->passphrase;
 		ssid_len = strlen((const char *)ssid);
 		password_len = ap_connect_config->passphrase_length;
-		key_id = 1; //Foucus
+		key_id = 0;

```

Comment:
you can initialize with zero at declaration, as nowhere it is getting modified or assigned different values

Reply:
I agree with you, thanks